ico_template.py

```diff
@@ -47,76 +46,59 @@ def Main(operation, args):
 
         # Otherwise, we need to lookup the assets and determine
         # If attachments of assets is ok
-        attachments = get_asset_attachments()  # type:Attachments
-
-        storage = StorageAPI()
-
-        crowdsale = Crowdsale()
-
-        return crowdsale.can_exchange(token, attachments, storage, True)
-
-
-    elif trigger == Application:
-
-        if operation != None:
+        attachments = get_asset_attachments()
+        return False
```

Comment:
why is this false shouldn't it be the statement below?

Reply:
That is correct!